_ml-commons-plugin/connectors.md

```diff
@@ -35,35 +74,21 @@ PUT /_cluster/settings
     "persistent": {
         "plugins.ml_commons.trusted_connector_endpoints_regex": [
             "^https://runtime\\.sagemaker\\..*\\.amazonaws\\.com/.*$",
```

Comment:
Copy and paste the value from https://github.com/opensearch-project/ml-commons/blob/2.9/plugin/src/main/java/org/opensearch/ml/settings/MLCommonsSettings.java#L122C14-L124